### PR TITLE
docs(kit): updated ecopasskit adoption view, dpp aspect added, business arch updated and other document changes

### PR DESCRIPTION
**Pull request by matbmoser**
## Why do we create this PR:
There is the requirement to update the Eco Pass KIT Adoption View

## What do we what to do in this PR
We want to update the adoption view, and add the necessary aspects and updated diagrams.

### Added 
- Added EcoPass KIT Logo
- Added `Digital Product Pass v3.0.0` aspect and explanations
  
### Updated
- Updated battery pass model to latest version `v4.0.0`
- Updated Business Architecture Diagram
- Updated all legacy diagrams to editable svgs -> TRG 1.04

## Content Review PRs:
- https://github.com/eclipse-tractusx/eco-pass-kit/pull/23
- https://github.com/eclipse-tractusx/eco-pass-kit/pull/28
- https://github.com/eclipse-tractusx/eco-pass-kit/pull/27

## Related PRs:
This is the first PR of the EcoPass KIT v2.0.0 Update

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs-kits/kits/Eco_Pass_KIT/changelog.md [hunk 1]
```diff
@@ -17,13 +29,14 @@ title: Changelog
...
+- SPDX-FileCopyrightText: 2023, 2024 Robert Bosch GmbH
+- SPDX-FileCopyrightText: 2023, 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+- SPDX-FileCopyrightText: 2023, 2024 T-Systems International GmbH
+- SPDX-FileCopyrightText: 2023, 2024 SAP SE
+- SPDX-FileCopyrightText: 2023, 2024 CGI Deutschland B.V. & Co. KG
+- SPDX-FileCopyrightText: 2023, 2024 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V. für ihre Institute IPK und IPK
+- SPDX-FileCopyrightText: 2023, 2024 BASF SE
+- SPDX-FileCopyrightText: 2023, 2024 Henkel AG & Co. KGaA
+- SPDX-FileCopyrightText: 2023, 2024 Contributors to the Eclipse Foundation
+- Source URL: https://github.com/eclipse-tractusx/eco-pass-kit
```
Comment (SebastianBezold) [inline docs-kits/kits/Eco_Pass_KIT/changelog.md:42]: Do you want to keep this link to the archived repo? I think pointing here would be better

File: docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md [hunk 2]
```diff
@@ -21,16 +23,14 @@ The EcoPass KIT envisions a future where sustainability is at the core of every
...
+All described specifications in the KIT are based on [Catena-X standards](https://catena-x.net/de/standard-library) such as the Asset Administration Shell, SSI and decentral Digital Twin Registry. They refer to other Catena-X KITs like the [Connector KIT]/<https://eclipse-tractusx.github.io/docs-kits/kits/tractusx-edc/docs/kit/adoption-view/Adoption%20View/> (EDC), [Data Chain KIT](https://eclipse-tractusx.github.io/docs-kits/category/data-chain-kit/) (Item Relationship Service = IRS) and [Business Partner KIT](https://eclipse-tractusx.github.io/docs-kits/category/business-partner-kit/) to ensure interoperability and data sovereignty according to IDSA and Gaia-X principles.
```
Comment (SebastianBezold) [inline docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md:26]: The link to the connector KIT is no longer proper markdown. Is this intended?

**Review by SebastianBezold — COMMENTED** [review]
I guess the source URLs in all of your copyright and license notices should be changed

**Comment by FaGru3n** [comment]
Friendly Reminder for the pictures and diagrams https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-4/#drawio-example within this Pull-Request

**Comment by matbmoser** [comment]
> Friendly Reminder for the pictures and diagrams https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-4/#drawio-example within this Pull-Request

The diagram included was made with PowerPoint originally. I need to attach the PowerPoint file then?

**Comment by FaGru3n** [comment]
> > Friendly Reminder for the pictures and diagrams https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-4/#drawio-example within this Pull-Request
> 
> The diagram included was made with PowerPoint originally. I need to attach the PowerPoint file then?

I would suggest the the diagrams in another file format (`.svg`. `.puml`, `.mermaid`) and a little desricption how to modify and update the diagram in the furture.

File: docs-kits/kits/Eco_Pass_KIT/changelog.md (hunk 1 above)
Comment (matbmoser) [reply docs-kits/kits/Eco_Pass_KIT/changelog.md:42]: Maybe to point to digital product pass

**Comment by matbmoser** [comment]
> > > Friendly Reminder for the pictures and diagrams https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-4/#drawio-example within this Pull-Request
> > 
> > 
> > The diagram included was made with PowerPoint originally. I need to attach the PowerPoint file then?
> 
> I would suggest the the diagrams in another file format (`.svg`. `.puml`, `.mermaid`) and a little desricption how to modify and update the diagram in the furture.

@FaGru3n I am with you, but I also have no capacity now to translate a document from PowerPoint format into Draw.io

This diagram was already included in the KIT, the maximum I can do is to try to add it in .svg.

**Comment by matbmoser** [comment]
I tried to add it in svg but the fonts look really bad

**Comment by FaGru3n** [comment]
> This diagram was already included in the KIT, the maximum I can do is to try to add it in .svg.

but then you will miss the [TRG 1-04](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-4/#drawio-example) that to have editable and maintainable diagrams with a description how to maintain these files.

We as committers aligned on that, so i get your point that this is a bit annoying, but with this will secure the maintainability for upcomming releases.

Thanks in advance.

File: docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md (hunk 2 above)
Comment (matbmoser) [reply docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md:26]: Good that you spotted that! I will fix it

File: docs-kits/kits/Eco_Pass_KIT/changelog.md (hunk 1 above)
Comment (matbmoser) [reply docs-kits/kits/Eco_Pass_KIT/changelog.md]: Fixed

**Review by saudkhan116 — APPROVED** [review]
As already discussed with @matbmoser, the requested changes are now resolved, Anything looks fine to me.

**Review by dsrparracho — APPROVED** [review]

**Review by maxim-bo — APPROVED** [review]

**Review by LuisRickert — APPROVED** [review]

**Review by matbmoser — COMMENTED** [review]
I found some old diagrams that are still not transfered to svg I am transfering them now

**Review by matbmoser — COMMENTED** [review]
It was quite a lot of work, but I could translate the diagrams into editable diagrams

File: docs-kits/kits/Eco_Pass_KIT/changelog.md [hunk 3]
```diff
@@ -2,6 +2,18 @@
 title: Changelog
 ---
 
+## [released]
+## [2.0.0] - 2024-01-12
```
Comment (maximilianong) [inline docs-kits/kits/Eco_Pass_KIT/changelog.md:6]: Version needs to be changed to 1.0.0 and date needs to be adjusted to the release date: 2024-03-08

**Review by maximilianong — CHANGES_REQUESTED** [review]

**Review by maximilianong — CHANGES_REQUESTED** [review]
If you have standards, please add the corresponding and relevant version to it.

File: docs-kits/kits/Eco_Pass_KIT/changelog.md (hunk 3 above)
Comment (matbmoser) [reply docs-kits/kits/Eco_Pass_KIT/changelog.md:6]: done

**Comment by matbmoser** [comment]
> If you have standards, please add the corresponding and relevant version to it.

I the PR [#633](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/633) we are referencing all the standard we are following. The reference to the standard library was added so the latest standard can be checked:

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/f8f100ce1c7e19f60d82ea16130891f06c803c5a/docs-kits/kits/Eco_Pass_KIT/page-software-operation-view.md#L15-L24

If you refer to the latest Digital Product Pass Triangle and the Digital Product Pass Datamodel Standard. This ones are still not available for the open source community however yes the identifications can be added. Just not know if the Adoption View is the correct place to do it @LuisRickert?

**Comment by matbmoser** [comment]
@maximilianong  done

**Review by saudkhan116 — APPROVED** [review]

**Review by maximilianong — APPROVED** [review]

**Pull request merged**